core/src/components/item/item.ios.scss

```diff
@@ -204,18 +202,10 @@
   @include margin(10px, 8px, 10px, 0);
 }
 
-
 // iOS Stacked & Floating Inputs
 // --------------------------------------------------
 
 :host(.item-label-floating),
 :host(.item-label-stacked) {
   --min-height: 68px;
 }
```

Comment:
Was this missed? The corresponding styles in MD have been removed in this PR.

I was a little confused about why the `item-label-` styles are being removed in this PR, but it looks like the class hasn't been on item for a while anyway, so we're just removing the remaining styles now since it's convenient? Is that accurate?

Reply:
No, these should stay. These have to do with `ion-label` position property. We may need to discuss if the floating/stacked/fixed variants are still needed on `ion-label`, but the behaviors were not deprecated originally so I kept them in.